apps/approval/views.py

```diff
@@ -70,6 +68,20 @@ def create_fos_application(request):
     raise Http404
 
 
+def get_expiry_date(started_year, length_of_fos):
+    # Expiry dates should be 15th September, so that we have time to get new lists from NTNU
+    new_expiry_date = datetime.date(
+        started_year, 9, 16) + datetime.timedelta(days=365*length_of_fos)
+    # Expiry dates in the past sets the expiry date to next september
+    if new_expiry_date < datetime.date.today():
```

Comment:
Use `django.utils.timezone` instead of `datetime.datetime`. :-) That takes care of all timezone stuff (and DST/Summer time).